01week/resume/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<header>
+  <head>
+    <title>Mary Heppenstall's Resume</title>
+      <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+<h1>Mary Heppenstall</h1>
+</header>
+  <p>PROFESSIONAL EXPERIENCE</p>
+    <nav>KSM </nav>
+      <article>Supervisor, Media Platforms</article>
+      <article>  Austin, Texas </article>
+      <article> Jan 2017 – Present </article>
+          <li>Traffic and optimize all self-service digital buys for KSM South’s client base including social, programmatic display, programmatic video, and search advertising</li>
```

Comment:
don't forget that <li> tags, or "list items", go inside of a <ul> (unordered list) or <ol> (ordered list) tag.